source/extensions/filters/network/redis_proxy/command_splitter_impl.h

```diff
@@ -76,12 +76,16 @@ class SplitRequestBase : public SplitRequest {
   static void onWrongNumberOfArguments(SplitCallbacks& callbacks, const Common::Redis::RespValue& request);
   void updateStats(const bool success);
 
-  SplitRequestBase(CommandStats& command_stats, TimeSource& time_source)
+  SplitRequestBase(CommandStats& command_stats, TimeSource& time_source, bool latency_in_micros)
       : command_stats_(command_stats) {
-    command_latency_ms_ = std::make_unique<Stats::Timespan>(command_stats_.latency_, time_source);
+    if (latency_in_micros) {
+      command_latency_ = std::make_unique<Stats::Timespan<std::chrono::microseconds>>(command_stats_.latency_, time_source);
```

Comment:
Formatting is busted here, please remove the trailing slash here in the macro: https://github.com/envoyproxy/envoy/pull/6285/files#diff-d6b0a3c7951ede08ff87a40356019233R49 (this is preventing clang-format from turning back on)

Reply:
@mattklein123 I am not clear about which macro and 'trailing slash' you are referring to. The link is the PR diff. Could you please provide more details. I will fix it.

Reply:
Remove the continuation slash on this line: https://github.com/envoyproxy/envoy/blob/00eb0de116bdea17b7098ef1e4bef1843d14cc32/source/extensions/filters/network/redis_proxy/command_splitter_impl.h#L49

Reply:
Done